plutus-metatheory/src/Type/RenamingSubstitution.lagda.md

```diff
@@ -229,7 +252,7 @@ variable
   σ σ' : Sub Φ Ψ
 ```
 
-Extending a type substitution — used when going under a binder.
+Extending a type substitution — used when going under a binder. (This is called `lifts` in the [paper](https://ci.iog.io/build/1230848/download/1/paper.pdf#page=8) ).
```

Comment:
How do we keep this link up-to-date?

Reply:
Is there a DOI link somewhere?

Reply:
Our local copy is the best version as it includes a fix